tests/basic/sharedv4_svc_test.go

```diff
@@ -1307,6 +1309,163 @@ var _ = Describe("{Sharedv4SvcFunctional}", func() {
 	})
 })
 
+// This test runs a ML Workload that predicts Housing rents. It contains one pod that preprocesses model
+// and prepares training data. One pod runs continuously and keeps on retraining the model every 5 mins.
+// The other pods that are deployments run once every 5 mins and read the model, do some prediction and
+// writes their predictions to shared volume on which model is saved. Retraining pod picks this data up
+// and again retrains the model. This is a simple feed-forward neural network model.
+var _ = Describe("{CreateMlWorkloadOnSharedv4Svc}", func() {
+	ns := "ml-workload-ns"
+	appContexts := make([]*scheduler.Context, 0)
+	prereqContexts := make([]*scheduler.Context, 0)
+	taskName := "prepare-ml-workload"
+	totalMlDeps := 5
+	totalRunTime := 5
+	log.Infof("Original App list : %v", Inst().AppList)
```

Comment:
better move the line 1324 and 1325 inside justbefore each, else it will print even though this test is not run